packages/sirius-web/backend/sirius-web-application/src/main/resources/schema/siriusweb.graphqls

```diff
@@ -1,9 +1,17 @@
 extend type Viewer {
   project(projectId: ID!): Project
-  projects(after: String, before: String, first: Int, last: Int): ViewerProjectsConnection!
+  projects(after: String, before: String, first: Int, last: Int, filter: ProjectFilterInput): ViewerProjectsConnection!
   projectTemplates(page: Int!, limit: Int!): ViewerProjectTemplatesConnection!
 }
 
+input ProjectFilterInput {
```

Comment:
`ProjectFilter` keep the naming convention XxxInput for mutations, this is just an argument like other arguments.

Reply:
Done